pkg/scheduler/util/utils.go

```diff
@@ -91,13 +92,9 @@ func GetEarliestPodStartTime(victims *api.Victims) *metav1.Time {
 // SortableList is a list that implements sort.Interface.
 type SortableList struct {
 	Items    []interface{}
-	CompFunc LessFunc
+	CompFunc heap.LessFunc
```

Comment:
nope :)
Either `LessFunc` comes from a common package, or each package defines their own. A private alias should be enough.

Reply:
Changes as per recommendation